docs/index.md

```diff
@@ -334,7 +335,11 @@ through this `LocalEnsemble` object.
 Types of resources
 ------------------
 
-Currently there are six types of resources in bigml.com:
+Currently these are the types of resources in bigml.com:
+
+- **external connectors** Contain the information to connect to an external
+database manager. They can be used to upload data to BigML to build `sources`.
+These resources are handled through `bigml.ExternalConnector`.
 
 - **sources** Contain the data uploaded from your local data file after
```

Comment:
also add external sources here : sources** Contain the data uploaded from your local data file or retrieved from an external connector